lua/lvim/core/nvimtree.lua

```diff
@@ -67,6 +73,14 @@ function M.config()
         cmd = "trash",
         require_confirm = true,
       },
+      actions = {
+        change_dir = {
+          global = false,
+        },
+        open_file = {
+          quit_on_open = false,
+        },
```

Comment:
```suggestion
        },
      	window_picker = {
	    enable = false,
	    chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890",
	    exclude = {} -- not sure if we need to define this or not
	 },
```
This has replaced `disable_window_picker = 0`. You can keep it enabled if you'd like.